includes/reader-activation/class-reader-activation.php

```diff
@@ -1077,76 +1078,32 @@ public static function render_auth_form( $is_inline = false ) {
 				<?php endif; ?>
 				<div class="<?php echo \esc_attr( $class( 'content' ) ); ?>">
 					<form method="post" target="_top">
-						<div data-action="pwd link register">
+						<div class="<?php echo \esc_attr( $class( 'header' ) ); ?>">
+							<h2><?php _e( 'Sign In', 'newspack-plugin' ); ?></h2>
+						</div>
+						<div data-action="signin register">
 							<?php self::render_third_party_auth(); ?>
 						</div>
 						<input type="hidden" name="<?php echo \esc_attr( self::AUTH_FORM_ACTION ); ?>" value="1" />
 						<?php if ( ! empty( $referer['path'] ) ) : ?>
 							<input type="hidden" name="referer" value="<?php echo \esc_url( $referer['path'] ); ?>" />
 						<?php endif; ?>
 						<input type="hidden" name="action" value="pwd" />
-						<div class="<?php echo \esc_attr( $class( 'have-account' ) ); ?>">
-							<a href="#" data-action="pwd link" data-set-action="register"><?php \esc_html_e( "I don't have an account", 'newspack-plugin' ); ?></a>
-							<a href="#" data-action="register" data-set-action="pwd"><?php \esc_html_e( 'I already have an account', 'newspack-plugin' ); ?></a>
-						</div>
-						<div class="<?php echo \esc_attr( $class( 'header' ) ); ?>">
-							<h2><?php _e( 'Sign In', 'newspack-plugin' ); ?></h2>
-						</div>
 						<p data-has-auth-link>
 							<?php _e( "We've recently sent you an authentication link. Please, check your inbox!", 'newspack-plugin' ); ?>
 						</p>
 						<p data-action="pwd">
-							<?php
-								echo wp_kses_post(
-									sprintf(
-										// Translators: %s is the link to sign in via magic link instead.
-										__( 'Sign in with a password below, or %s.', 'newspack-plugin' ),
-										'<a href="#" data-set-action="link">' . __( 'sign in using your email', 'newspack-plugin' ) . '</a>'
-									)
-								);
-							?>
-						</p>
-						<p data-action="link">
-							<?php
-								echo wp_kses_post(
-									sprintf(
-										// Translators: %s is the link to sign in via password instead.
-										__( 'Get a code sent to your email to sign in, or %s.', 'newspack-plugin' ),
-										'<a href="#" data-set-action="pwd">' . __( 'sign in using a password', 'newspack-plugin' ) . '</a>'
-									)
-								);
-							?>
+							<strong>
+								<?php esc_html_e( 'Enter your password', 'newspack-plugin' ); ?>
+							</strong>
 						</p>
 						<p data-action="otp">
 							<strong>
 								<?php esc_html_e( 'Enter the code sent to your email.', 'newspack-plugin' ); ?>
 							</strong>
 						</p>
 						<input type="hidden" name="redirect" value="<?php echo \esc_attr( $redirect ); ?>" />
-						<?php if ( isset( $lists ) && ! empty( $lists ) ) : ?>
```

Comment:
Now that `$lists` is not used anymore, assignment to this variable on lines 1057-1062 is not necessary.

Reply:
Good catch! Removed in 24b72348d9b35df2924c876c8514429f17f97d49